lib/schema.js

```diff
@@ -19,6 +19,132 @@ class Schema {
         this.time = data.time || new Date().getTime();
     }
 
+    /**
+     * Initializes the Schema class
+     * @param {string} name
+     * @returns {string} SKU
+     */
+    getSkuFromName(name) {
+        name = name.toLowerCase();
+        const item = {
+            defindex: null,
+            quality: null,
+            craftable: false
+        };
+
+        if (name.includes('Strange Haunted'.toLowerCase())) {
+            item.quality = 13;
+            item.quality2 = 11;
+        } else {
```

Comment:
Should also do for `Strange Genuine`, `Strange Vintage` (except if "Strange Vintage Tyrolean" but not "Strange Vintage Vintage Tyrolean", and "Strange Vintage Merryweather" but not "Strange Vintage Vintage Merryweather"), and `Strange Unique`

![image](https://user-images.githubusercontent.com/47635037/111034816-4d0fd500-8452-11eb-83e5-0623eb34705e.png)

Reply:
Should be fixed for every combination of strange + <quality> are there any other combinations ?

Reply:
I guess Strange is done.